VSEmbed.DemoApp/VSEmbed.DemoApp.csproj

```diff
@@ -162,6 +162,10 @@
     <Reference Include="Microsoft.VisualStudio.Shell.TreeNavigation.HierarchyProvider, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\lib\Microsoft.VisualStudio.Shell.TreeNavigation.HierarchyProvider.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.Shell.UI.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
I had to add this to `DemoApp` because on my laptop it wasn't getting copied to the output directory. I see Shell.UI.Internal is GAC-ed on my laptop so maybe that's why it wasn't getting copied? I'll double check at work on Tuesday.